waspc/data/Generator/templates/sdk/wasp/client/router/types.ts

```diff
@@ -32,7 +32,7 @@ type RoutesDefinition = {
   }
 }
 
-type BuildFn = (params: unknown) => string
+type BuildFn = (params: never) => string
```

Comment:
This was throwing errors after enabling [strictFunctionTypes](https://www.typescriptlang.org/tsconfig/#strictFunctionTypes).

If we wanted to say "all `build` fields in route definitions must be a subtype of this function", then `never` is the correct argument type.

Reply:
This is fine 👍 Thanks for making it correct!